.github/workflows/conformance.yml

```diff
@@ -1,49 +1,40 @@
 name: Conformance Testing
 
 on:
-  push:
-    branches:
-      - main
-      - release-*
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+*"
-  pull_request:
-  schedule:
-    - cron: "0 4 * * *" # run every day at 4am UTC
```

Comment:
We're losing this nightly run

Reply:
Oh I keep forgetting that we don't run a nightly CI job...should we add it?

Reply:
The goal is just to run the conformance tests nightly, so if that now means we run the full CI job, then I guess that's how we do it.

Reply:
Although the current conformance job uses the trigger to set some variables on the nightly run...that is now changing unless we can pass flag to it from the CI job.

Reply:
We check if `${{ github.event_name == 'schedule' }}` in a few places, but this will probably no longer be true if it's always triggered via workflow call

Reply:
I think it doesn't hurt to run the entire pipeline every night...
The event name will still work, so we don't need to change anything.